eng/ci/templates/official/jobs/run-coldstart.yml

```diff
@@ -0,0 +1,129 @@
+parameters:
+- name: description
+  type: string
+- name: functionAppName
+  type: string
+- name: instanceId
+  type: string
+- name: os
+  type: string
+  default: Windows
+  values:
+    - Windows
+    - Linux
+
+jobs:
+- job: ${{ parameters.functionAppName }}_${{ parameters.instanceId }}
+  displayName: ${{ parameters.description }} ${{ parameters.instanceId }}
+  pool:
+    name: 1es-pool-azfunc-benchmarking-large
```

Comment:
We may want a smaller pool with more agents available if we will be running this with like 20 machines at a time. Unless you plan on making these run in sequence? Will take longer but will use far less VMs overall.

Reply:
You meant large pool (size) with small VMs. correct? The current images used(-sd4) are 4 core machines. I am creating 2 core ones now and will push an iteration later today after testing on that version.

Reply:
Pushed an iteration which uses images with 2 cores.

Reply:
The image does not control core count, the pool does. `1es-pool-azfunc-benchmarking-large` is an 8 core pool (standard D8as v5)

Reply:
Oh, I didn't realize that. Yea, I updated the images now to be the originals (1es-ubuntu-22.04-benchmark-runner-vanilla, 1es-windows-2022-benchmark-runner-vanilla) and update the pool size to d4.